TrumpTracker/trumptracker.github.io
# Progress of policy now displays in search

Fixes #39 
If the search text appears in the text of the current `<td>` element being checked; do what was done before but then grab the `<tr>` it was residing in (its parent) and grab its class.

If the class is anything but empty, add the relevant class to the `<li>` element being appended to search results.

## To Test
- In `data.yaml` change the status of some policies to either `broken` or `achieved`
- Search for the policy in the search bar
- Policy should appear with success class if policy was set to achieved, danger class if broken or normal otherwise
